tasks/copy.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
   // Copy images to /dist directory
   copyImages() {
     dutil.logMessage("copyImages", "Copying images to /dist/img");
-    return src(["packages/**/src/img/**/*{png,jpg,gif,webp,svg,ico}"])
+    return src(["packages/**/src/img/**/[!_]*.{png,jpg,gif,webp,svg,ico}"])
```

Comment:
Thoughts on moving processed images to a `dist` directory and keep unprocessed in `src`?

It would remove ambiguity in this glob and what the underscore means.

Reply:
Dist directory issue has been moved to #4886. Resolving this thread.